CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
```

Comment:
Resolving the items also takes ~30 seconds:

![image](https://user-images.githubusercontent.com/4947061/211203956-3c683eb1-bbf1-4067-8259-26d58722bee5.png)